hack/ci-deploy-release.sh

```diff
@@ -33,6 +33,12 @@ fi
 
 ARTIFACTS="${ARTIFACTS:-$( mktemp -d )}"
 
+if [ -z "${DEPLOY_DIR+x}" ]; then
```

Comment:
I find this easy to mix with the actually `/deploy/` - `ARTIFACTS_DEPLOY_DIR`?

Reply:
ok, changed. FYI though, this is how it's set up in the regular deploy script

Reply:
ok, I haven't checked that, then the consistency would be a better reason than my nit.